cmd/hyperkube/hyperkube.go

```diff
@@ -123,8 +123,8 @@ func (hk *HyperKube) Run(args []string) error {
 	command := args[0]
 	baseCommand := path.Base(command)
 	serverName := baseCommand
+	args = args[1:]
 	if serverName == hk.Name {
-		args = args[1:]
```

Comment:
Ironically enough, this was actually a fix to make softlinking work better :) I've tested it.

There's no reason to pass the command arg to any of the Server Run blocks . In particular, since `kubectl` is now a Cobra command, it requires that if `args[0]` is not a flag, it match the command name exactly. In the case of a softlink, it's possible thats `args[0]` will be for example `/usr/bin/kubectl`, and the Cobra command will complain. This removes that possibility.